tests/unit/test_story.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from entities.schemas import Node
+from utilities.exceptions import DynamoDBError
+
+
+@pytest.mark.usefixtures("mock_db_session", "test_data")
+class TestStoryDataStore:
+    def test_get_story(self, mock_db_session, test_data):
+        story = mock_db_session.get_story("test_id")
+        print(story.id, test_data)
+        assert story.id == test_data["id"]
+        assert story.root == test_data["root"]
+
+    def test_get_not_exist_story(self, mock_db_session):
+        story = mock_db_session.get_story("not_exist_id")
+        assert story is None
```

Comment:
This should work without class
```suggestion
def test_get_not_exist_story(mock_db_session):
    story = mock_db_session.get_story("not_exist_id")
    assert story is None
```